RELEASES.md

```diff
@@ -53,18 +156,39 @@ was incorrectly sign-extended. Now you will always get a value that only ever ha
     get things done for now.
 
 
+## Better Stackwalking
+
+* We now handle noreturn callees properly in the stack scanning heuristics
+    * When you call a noreturn function you are not obligated to have any code following the CALL instruction, so the implicitly pushed return pointer may be "one-past-the-end" of the function's code. Subtracting 1 from the return pointer is sufficient to handle this.
+* We no longer \~infinite loop when the stackpointer doesn't change on ARM64
+    * Same issue we had on ARM but we forgot to port the fix to the ARM64 backend
+* Many fixed overflows that were \~benign but would assert and crash debug builds
+
+
+## More information in ProcessState
+
+* **BREAKING CHANGE**:`SystemInfo::os_version` has been split into two fields.
+    * `os_version`: "5.1.2600"
+    * `os_build`: "Service Pack 2"
+    * system_info.os_ver in the JSON schema still contains the combined value.
+    * The combined value can be obtained with `SystemInfo::format_os_version`
+
+* Threads (CallStacks) now include their original thread_id
+
+
+
 ## Better Errors
 
-When minidump-stackwalk encounters a fatal error (basically only happens for empty
-or Effectively Empty minidumps), it will include an error type before the message.
+* More detailed formatting of OS-specific errors 
+    * macOS EXC_GUARD
+    * macOS KERN_FAILURE
```

Comment:
There's also the changes to the Linux signals that go here. When printing out signals we omit the value of `si_code` when it's 0 like Breakpad did (because it's not interesting) and we print it out as a human readable code when it's non-zero (which is an improvement over what we had in Breakpad). See PR #431 for the details.